docs/resources/nat_gateway.md

```diff
@@ -49,6 +49,10 @@ The following arguments are supported:
 * `description` - (Optional, String) Specifies the description of the NAT gateway, which contain maximum of `512`
   characters, and angle brackets (<) and (>) are not allowed.
 
+* `ngport_ip_address` - (Optional, String, ForceNew) Specifies the private IP address of the NAT gateway.
+  The IP address is assigned by the VPC subnet.
```

Comment:
It is designated or allocated?

Reply:
It is can be spacified, if not specified, will be allocated.